src/igvfd/loadxl.py

```diff
@@ -25,9 +25,9 @@
     'human_donor',
     'rodent_donor',
     'treatment',
+    'tissue',
```

Comment:
why did this move?

Reply:
so primary_cell can point to tissue when loading the inserts

Reply:
that will break once we hook in the other direction (primary_cell -> tissue will work now, but (cell_line -> primary_cell will not). It should be handled in 
PHASE1_PIPELINES = {
    'user': [
        remove_keys('lab', 'submits_for'),
    ],
} instead

Reply:
@ian-whaling I think this detail about moving this logic to PHASES is what is left.  Do you need more clarity on this request or have you figured it out?